sklearn/metrics/tests/test_common.py

```diff
@@ -1201,3 +1203,47 @@ def test_no_averaging_labels():
             score_labels = metric(y_true, y_pred, labels=labels, average=None)
             score = metric(y_true, y_pred, average=None)
             assert_array_equal(score_labels, score[inverse_labels])
+
+
+@pytest.mark.parametrize(
+    'name', MULTILABELS_METRICS - NOT_SYMMETRIC_METRICS)
```

Comment:
Why do we need symmetry?

Reply:
It does not need it. This was a little too restrictive. 

This PR was updated to remove the "unnormalized_multilabel_confusion_matrix" from this test. It returns a matrix as a metric, which makes the result dependent on the order of the labels.